packages/big-design/src/components/ToggleButton/styled.tsx

```diff
@@ -0,0 +1,39 @@
+import { theme as defaultTheme } from '@bigcommerce/big-design-theme';
+import styled from 'styled-components';
+
+interface StyledToggleButtonProps {
+  selected: boolean;
+}
+
+export const StyledToggleButton = styled('button')<StyledToggleButtonProps>`
```

Comment:
Try extending from `StyleableButton` located in `components/Button/private.ts`. You'll be able to write less CSS and use some of the baked in features our `Button` component has.